### PR TITLE
Fixing error PHP8: Unknown named parameter $action

### PR DESCRIPTION
**Pull request by huubl**
Selecting a category in the Community Events submission form is broken in PHP8:

```
Unknown named parameter $action {"userId":2,"exception":"[object] (Error(code: 0): Unknown named parameter $action at [...]plugins/the-events-calendar/common/src/Tribe/Ajax/Dropdown.php:278)

[stacktrace]
#0 [...]/wp-includes/class-wp-hook.php(303): Tribe__Ajax__Dropdown->route('')
#1 [.../wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#2 [...]/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#3 [...]/wp-admin/admin-ajax.php(187): do_action('wp_ajax_tribe_d...')
#4 {main}
"}
```

P.S. It would come in handy if the option to change the PHP version is activated in a sandbox created with: https://theeventscalendar.com/demo/. Since support needs [my system information](https://theeventscalendar.com/knowledgebase/k/sharing-your-system-information/) and I rather not share data from client sites.

### CODE REVIEW

**Comment by bordoni** [comment]
Hi @huubl,

I will make sure we have this merged for our next release. Sorry for not catching this way earlier.

File: src/Tribe/Ajax/Dropdown.php [hunk 1]
```diff
@@ -275,7 +275,7 @@ public function route() {
 		if ( has_filter( $filter ) ) {
 			$data = apply_filters( $filter, [], $args->search, $args->page, $args->args, $args->source );
 		} else {
-			$data = call_user_func_array( [ $this, $args->source ], (array) $args );
+			$data = call_user_func_array( [ $this, $args->source ], array_values((array) $args) );
```
Comment (bordoni) [inline src/Tribe/Ajax/Dropdown.php:278]:
```suggestion
			$data = call_user_func_array( [ $this, $args->source ], array_values( (array) $args ) );
```

**Comment by huubl** [comment]
Any update on when this gets merged?

**Review by juanfra — DISMISSED** [review]
🚀

**Pull request merged**